src/containers/MyActivities.jsx

```diff
@@ -175,7 +175,7 @@ class MyActivities extends Component {
                         <ActivityCard
                           id={activity.id}
                           category={activity.category}
```

Comment:
Can we de-structure in the same format as the verifyActivities? For consistency purposes